frontend/src/components/TaskMapTask.module.scss

```diff
@@ -48,6 +48,10 @@
   &.loading:not(.dragging, .draggingOutside) {
     cursor: wait;
   }
```

Comment:
Looking at this, the loading cursor style should also be applied to all the cases where the `not-allowed` is (the group, and the handles), as it now doesn't seem to show consistently.